.travis.yml

```diff
@@ -23,22 +48,30 @@ addons:
     - g++-4.8
 env:
   matrix:
-  - CXX=g++-4.8
+  - EMBER_TRY_SCENARIO=default
+  - EMBER_TRY_SCENARIO=ember-release
+  - EMBER_TRY_SCENARIO=ember-2-8
   global:
-    secure: m3T8OBefKGIJLK1hogS6WvLXDxGSMX539d6Gazgd9D42Q/iF7ni+58fZM8akQOQMAzNssg82Yv29yS+GSudL68+jKHrzLviZ8gc77FwhLagEHOrxFZWPEdjNwNETkocXX03Rda+Bzo9ZaM+RA2jmJfcxxqu9U9rvjHgqNFyqeVLMrtI8BR0vIlyXEnk9lIroBy8Nw8xTFIQkaB3hJlPjbhnYHQSFlRnn8+gtcdu1ZrlGzm8uLHSacLLlcS0fF42tGeN54eGrgpMhhlOe1Kiu+eCIxnq96uoRWwKSfw+8WqidH/0CTcTaYirIXNdxPjUU3AyHeItmOG7TidAbrhfibu8S57OvVy5dL2cxixx8NRZBwmra0HEA0qEdP381K4Re6rJhJNL93B24SWw0UL7X2KmoFBjiPFYoom8VUm9v6rIe0YMtGYp9a6wBzI+Wx5hpdRQrWG9Rj2H6Sbnrfhl2SfMQwCPyHMpIxLfNAwsV4V6smX0w3MqmLHnzWdLj14k0ETOFVjOpcFxVeza/jo7XLW5FdpEgAxSZXDTI8qVMl2ViqeezTo9EKCHBylOEKIT3/usXbWpN5qAQWR7M/qyua31tWDrgsiNFpWncJQaFcRDF30bvULznR+GILEEk9ZwiBIgeGucsAkDEzQiPY62I8tRx7B26OK2X/sBPW09qNIk=
-before_deploy:
-- pr-bumper bump
+    - CXX=g++-4.8
+    - secure: m3T8OBefKGIJLK1hogS6WvLXDxGSMX539d6Gazgd9D42Q/iF7ni+58fZM8akQOQMAzNssg82Yv29yS+GSudL68+jKHrzLviZ8gc77FwhLagEHOrxFZWPEdjNwNETkocXX03Rda+Bzo9ZaM+RA2jmJfcxxqu9U9rvjHgqNFyqeVLMrtI8BR0vIlyXEnk9lIroBy8Nw8xTFIQkaB3hJlPjbhnYHQSFlRnn8+gtcdu1ZrlGzm8uLHSacLLlcS0fF42tGeN54eGrgpMhhlOe1Kiu+eCIxnq96uoRWwKSfw+8WqidH/0CTcTaYirIXNdxPjUU3AyHeItmOG7TidAbrhfibu8S57OvVy5dL2cxixx8NRZBwmra0HEA0qEdP381K4Re6rJhJNL93B24SWw0UL7X2KmoFBjiPFYoom8VUm9v6rIe0YMtGYp9a6wBzI+Wx5hpdRQrWG9Rj2H6Sbnrfhl2SfMQwCPyHMpIxLfNAwsV4V6smX0w3MqmLHnzWdLj14k0ETOFVjOpcFxVeza/jo7XLW5FdpEgAxSZXDTI8qVMl2ViqeezTo9EKCHBylOEKIT3/usXbWpN5qAQWR7M/qyua31tWDrgsiNFpWncJQaFcRDF30bvULznR+GILEEk9ZwiBIgeGucsAkDEzQiPY62I8tRx7B26OK2X/sBPW09qNIk=
+
+matrix:
+  fast_finish: true
+  allow_failures:
+  - env: EMBER_TRY_SCENARIO=ember-release
+
 deploy:
   provider: npm
   email: npm.ciena@gmail.com
   skip_cleanup: true
   api_key:
     secure: KP3qIhLdSto+b4GRx0RlJ5ISTPPVVo49doXIRPJxB/QwTG0uepdrWda7DQTdZuYR/YFW08bPXM7UIcT6IQIy7P0Mo/gERyM/hKKfjMuuTjR0DwG9McUW0v37uM4QJ6Uw/v8rg3GilFurrq15KWmiL2Wzcz8vS8G9Y9+4FZVZxMkqzSPsT6IKP3Y8HHKAs5Ueu7jLI7SaaQL+u8go+hG169FmdhLVw6ahOsFaEZCLJgFO1/s1rFL5FWWf0YT0NRDrGB1fiC41QWgnUaHoyvwx8tkdRY2nZoCb1yaw8F/IlzFsiHAUfWXRmihhtgISFY+SqMQlkAyFsSsTZnCKu8ln6aWyr+JEHPGJ2yrI70bbXaQPT9k/gI6oLXOv6KcVHki9t1bT4gnilM5QAgyi8bktRApeVnBcZNy5AH6rb0w669elSJ57PyaXdGV/ChjxY5WaAb7epPKwr/fSL/tMi7URgg2lyArmfRuxqOY3G1m4ff6FFD15pO5dPnkyejpefcwxq/80ssnHhoG5fjP7f0A1wGck9WXXB0+Rvz4I22GDL4g0q2oE+ioRl8a9hK0rmxgJCiTRmVQ9FhunN5lTNcFAzzA/Beg2GrY5vxC5TzDP6LpYwjJoEV9FIkmM+qtXK7zOt4/jWMqNhtwg7wuD9iGl+Qgk4stsKSp0f353vgCLXBo=
   on:
-    branch: master
-    tags: false
-after_deploy:
-- .travis/publish-gh-pages.sh
+    all_branches: true
+    condition: "$EMBER_TRY_SCENARIO = 'default'"
+    node: '6.9.1'
```

Comment:
Also need to make the changes here as well:
https://github.com/rox163/ember-frost-pick-list/blob/master/.travis/maybe-bump-version.sh#L3
https://github.com/rox163/ember-frost-pick-list/blob/master/.travis/maybe-publish-coverage.sh#L11